### DISCUSSION TITLE
Custom workflow

### DISCUSSION THREAD

**Discussion by OkayashiM** [YetiForceCompany/YetiForceCRM#17539] [How to]
Gents.... cannot trigger custom workflow. existing custom workflows works perfect but very basic one not. Tried many various scenarios. workflow is correctly registered in database, files privileges are OKEY. YF 6.5 
Any clue? Error is:

2024-07-10 12:36:31.0171 [error][] - call_user_func(): Argument #1 ($callback) must be a valid callback, class "calculateWordCount" not found
TypeError: call_user_func(): Argument #1 ($callback) must be a valid callback, class "calculateWordCount" not found in /var/www/html/modules/com_vtiger_workflow/VTEntityMethodManager.php:47
Stack trace:
#0 /var/www/html/modules/com_vtiger_workflow/tasks/VTEntityMethodTask.php(36): VTEntityMethodManager->executeMethod(Object(Vtiger_Record_Model), 'AI_CustomWordCo...')
#1 /var/www/html/modules/com_vtiger_workflow/Workflow.php(229): VTEntityMethodTask->doTask(Object(Vtiger_Record_Model))
#2 /var/www/html/modules/Vtiger/handlers/Workflow.php(139): Workflow->performTasks(Object(Vtiger_Record_Model))
#3 /var/www/html/modules/Vtiger/handlers/Workflow.php(59): Vtiger_Workflow_Handler->performTasks(Object(App\EventHandler), Array)
#4 /var/www/html/app/EventHandler.php(501): Vtiger_Workflow_Handler->entityAfterSave(Object(App\EventHandler))
#5 /var/www/html/app/EventHandler.php(477): App\EventHandler->triggerHandler(Object(Vtiger_Workflow_Handler))
#6 /var/www/html/modules/Vtiger/models/Record.php(556): App\EventHandler->trigger('EntityAfterSave')
#7 /var/www/html/modules/Vtiger/actions/Save.php(109): Vtiger_Record_Model->save()
#8 /var/www/html/modules/Vtiger/actions/SaveAjax.php(25): Vtiger_Save_Action->saveRecord(Object(App\Request))
#9 /var/www/html/include/main/WebUI.php(188): Vtiger_SaveAjax_Action->process(Object(App\Request))
#10 /var/www/html/index.php(26): Vtiger_WebUI->process(Object(App\Request))
#11 {main}

**Comment by OkayashiM** [comment]
I will appreciate if someone can provide super basic working CW updating some field in whatever custom module. Thing is I probably missing some call to dependent code... thanks in advance

**Reply by apcloic** [reply]
Hi,

It looks like you have some bad code in your custom workflow. 

You will find basic examples here :
#13231

Regards,
